### DISCUSSION TITLE
If I want to use Tina with SvelteKit, should I look somewhere else?

### DISCUSSION THREAD

**Discussion by planetaska** [tinacms/tinacms#3858] [General]
Been a lover of Forestry.io for its simplicity, so when I heard Tina is entering 1.0, I was very excited. But after looking at the tutorials and docs, I find there is one big caveat for us to use Tina - the React requirement for visual editing. Other next-gen CMS with visual editing such as Storyblok doesn't really care what we use for our site. But with Tina it seems some framework specific config will be required. And without an example it is worrisome we will just be wasting time trying to make Tina work.

And from an old [issue](https://github.com/tinacms/tinacms/issues/331) it seems the Svelte support is not being worked on by the team. So the question is simple: should I ignore Tina for now until it is ready for other frameworks?

**Comment by kldavis4** [comment]
I believe @jamespohalloran [answer](https://github.com/tinacms/tinacms/issues/331#issuecomment-1450154229) in the linked issue is still the current state of things with SvelteKit:

> Svelte should work with the basic editor by following this guide: https://tina.io/docs/frameworks/other/
> Visual editing on the other hand will be looked at on a framework-by-framework basis, and doesn't currently have a timeline for Svelte.

So basic editor support is available, but contextual (visual) editing isn't supported. Anything to add @jamespohalloran ?

**Reply by jamespohalloran** [reply]
That's right! I think we'll likely add a sveltekit guide once we get a chance to dogfood more, but the generic framework doc should work in the meantime: https://tina.io/docs/frameworks/other/

**Reply by planetaska** [reply]
Got it. I think we can wait a bit longer for things to sort out. Really hope to see Tina fully working with SvelteKit. Thank you for the answer!

**Reply by SrGeneroso** [reply]
Bump on this. I want to use svelte and I'll follow the guide. I'm not a great programmer but may I ask, maybe it help others. How could I help to add another frameworks? Obvious place to look first is the examples folder. What are some considerations I should have if I want to PR a sveltkit example?
